packages/admin/cms-admin/src/dashboard/DashboardHeader.tsx

```diff
@@ -45,13 +45,25 @@ const Root = styled("div")<RootProps>`
                 background-image: url(${backgroundImageUrl["2x"]});
             }
         `}
+
+    ${({ theme }) => theme.breakpoints.down("sm")} {
+        height: 230px;
+    }
```

Comment:
We generally use a "mobile-first" approach for styling. 
Could you please make `230px` the default and change it to `300px` above `sm`?